### PR TITLE
incorrect constructN REPL

### PR DESCRIPTION
**Pull request by jdkschang**
fixed incorrect wrapping parens causing the last parameter to not get called in R.map

### CODE REVIEW

**Comment by kedashoe** [comment]
Hi @jdkschang , thanks for the PR! Could you update it to keep the same indentation as before?

**Comment by jdkschang** [comment]
@kedashoe you got it!

**Comment by kedashoe** [comment]
Is it too far indented now? Or are tabs being inserted? Not trying to be difficult, I just can't really tell what is actually changed vs all the changes in the diff because of the spacing. Can you play with the spacing a little more and run `git diff` locally until just the lines you want changed are showing up in the diff?

**Comment by jdkschang** [comment]
Sorry for any inconsistent pull requests. I am new to this, but hopefully the last couple of commits will reflect the fix more explicitly than my first couple tries.

Cheers

**Comment by kedashoe** [comment]
> Sorry for any inconsistent pull requests. I am new to this, but hopefully the last couple of commits will reflect the fix more explicitly than my first couple tries.
> 
> Cheers

Not a problem @jdkschang , thanks for sticking with it :) It's looking good, could you please squash the commits?

**Comment by davidchambers** [comment]
Hmm. As of c8d563fed7c6168d2028e0ed48fc267770670208 the only change I see is the addition of an unnecessary semicolon.

**Comment by kedashoe** [comment]
> Hmm. As of c8d563f the only change I see is the addition of an unnecessary semicolon.

huh ya seems like github is getting confused? Sometimes the "Files Changed" tab shows the full diff and sometimes just that one semicolon. When I click "View" on the file the changes are there. And pulling it down locally looks fine..

File: src/constructN.js [hunk 1]
```diff
@@ -21,8 +21,7 @@ var nAry = require('./nAry');
  *      // Variadic Constructor function
  *      function Salad() {
  *        this.ingredients = arguments;
- *      }
- *
+ *      };
```
Comment (davidchambers) [inline src/constructN.js]: Remove this semicolon, please. :)

**Comment by jdkschang** [comment]
@davidchambers The changes are actually in reference to the tag version `v0.24.1` where the example has the R.map incorrectly wrapped as shown [here](https://goo.gl/tV66fb).

I'm guessing the problem stems from my inexperience in initiating a pull request without following the proper channels (?). I've referred to the `CONTRIBUTING.md` but have little to no knowledge of the protocols regarding pull requests for a specific tag version: `v0.24.1`.

@kedashoe commits have been squashed

File: src/constructN.js (hunk 1 above)
Comment (jdkschang) [inline src/constructN.js]: roger

**Comment by kedashoe** [comment]
As far as I can tell it's just github. @davidchambers ?

**Comment by davidchambers** [comment]
```console
$ git --no-pager diff master..jdkschang/constructN
diff --git a/src/constructN.js b/src/constructN.js
index 606a240..df5c820 100644
--- a/src/constructN.js
+++ b/src/constructN.js
@@ -22,7 +22,6 @@ var nAry = require('./nAry');
  *      function Salad() {
  *        this.ingredients = arguments;
  *      }
- *
  *      Salad.prototype.recipe = function() {
  *        var instructions = R.map(ingredient => 'Add a dollop of ' + ingredient, this.ingredients);
  *        return R.join('\n', instructions);
```

**Comment by kedashoe** [comment]
Looks like this was already done here: https://github.com/ramda/ramda/pull/2216

So maybe you had branched from before that was merged, @jdkschang ?

**Comment by jdkschang** [comment]
Must be. Sorry for all the trouble I may have caused @kedashoe @davidchambers 😅. I was under the incorrect assumption that the [ramda docs](http://ramdajs.com/docs/) were representative of the current master branch.

**Pull request closed without merging**